test/mocks/fetchMock.js

```diff
@@ -25,7 +25,7 @@ const HELIX_ENV_JSON = {
     host: 'live.example.com',
   },
   project: 'Adobe Business Website',
-  contentSourceUrl: 'https://adobe.sharepoint.com/:f:/s/Dummy/Alk9MSH25LpBuUWA_N6DOL8BuI6Vrdyrr87gne56dz3QeQ',
+  mountpoints: 'https://adobe.sharepoint.com/:f:/s/Dummy/Alk9MSH25LpBuUWA_N6DOL8BuI6Vrdyrr87gne56dz3QeQ',
```

Comment:
Delete `HELIX_ENV_JSON` and add a mock for a `CONFIG_JSON` instead

Reply:
fixed in [e5b214b](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/e5b214b202d1e462982b19e00ef5de3a2696d022)